templates/correct.html

```diff
@@ -353,6 +349,20 @@ <h3>Tölfræði</h3>
       return fmt.replace(/\[bold\]/g, "<b>").replace(/\[~bold\]/g, "</b>");
    }
 
+   function getReferences(references) {
+      // Add links to references if available
+      var combo = " ";
+      $.each(references, function (rix, r) {
+         // reference r
+         // A few possible symbols:
+         // 👻🧚🦄🛎🌈⚡🎱👛🔔💎🔍🔎💡📖🔗🔬❓❔ℹ🇮🇸
+         combo += '<a href="' + r + '" target="_blank">🧚</a>';
```

Comment:
I acknowledge that this is cute but I suggest something more neutral and professional, such as a simple "play" icon (triangle).

Reply:
Also, think about white space around each reference icon. Could be implemented with a simple addition to the CSS, using margin-left and margin-right.

Reply:
Definitely, the current one was a placeholder until a decision was made. A play icon sounds good, what do you think about the book or light bulb in the list of possible symbols above?